.github/workflows/pr_test_build.yml

```diff
@@ -113,6 +113,8 @@ jobs:
           touch lib/.secrets.g.dart
           touch cw_evm/lib/.secrets.g.dart
           touch cw_solana/lib/.secrets.g.dart
+          touch cw_core/lib/.secrets.g.dart
```

Comment:
please remove anything related to creating secrets for cw_core and only add the `nanoNowNodesApiKey` in cw_nano's secrets

Reply:
if we remove the api key from cw_core how should we get the api key into
`cw_core/lib/node.dart`?

specifically these lines:
```
    if (uri.host == 'nano.nownodes.io') {
      headers['api-key'] = secrets.nanoNowNodesApiKey;
    }
```

Reply:
how was this handled for tron? does checking the node just not need the api key for tron for some reason?